test/functional/wallet_listsinceblock.py

```diff
@@ -291,5 +294,51 @@ def test_double_send(self):
             if tx['txid'] == txid1:
                 assert_equal(tx['confirmations'], 2)
 
+    def double_spends_filtered(self):
+        '''
```

Comment:
nit: could improve the code docs and formatting of the original author to current codebase conventions